testhiml/health/azure/test_himl.py

```diff
@@ -564,12 +603,17 @@ def test_invoking_hello_world_no_config(run_target: RunTarget, tmp_path: Path) -
         'body': 'print(f"The message was: {args.message}")'
     }
     extra_args = [f"--message={message_guid}"]
-    output = render_and_run_test_script(tmp_path, run_target, extra_options, extra_args, run_target == RunTarget.LOCAL)
     expected_output = f"The message was: {message_guid}"
     if run_target == RunTarget.LOCAL:
+        output = render_and_run_test_script(tmp_path, run_target, extra_options, extra_args,
+                                            run_target == RunTarget.LOCAL,
+                                            suppress_config_creation=run_target == RunTarget.AZUREML)
         assert expected_output in output
     else:
-        assert "Cannot glean workspace config from parameters, and so not submitting to AzureML" in output
+        with pytest.raises(ValueError) as e:
+            render_and_run_test_script(tmp_path, run_target, extra_options, extra_args, run_target == RunTarget.LOCAL,
+                                       suppress_config_creation=run_target == RunTarget.AZUREML)
```

Comment:
And here: suppress_config_creation is True. Is it clearer if explicitly set to True?

Reply:
Done